client/src/audio.js

```diff
@@ -1,6 +1,5 @@
 import * as Tone from 'tone';
 
-const MY_WALLET_ADDRESS = '0x963CFC0Bfb272BA9512621a677A31884c5c2A4DB'; // Vijay's Metamask wallet
```

Comment:
@vijayrudraraju, not sure if these values should stick around for future reference.  I went ahead and deleted them just in case.

Reply:
lol but is that the address that I can send @vijayrudraraju 's DAI to?

Reply:
Ha thanks